packages/convex-helpers/server/index.ts

```diff
@@ -12,7 +12,7 @@ import {
   PaginationResult,
 } from "convex/server";
 import { GenericId, Infer, ObjectType, Validator, v } from "convex/values";
-import { Expand } from "../index.js";
+import { Expand } from "..";
```

Comment:
I don't think this was intentional?

Reply:
thanks, was reverting some formatting changes